### PR TITLE
Allow blind logos

### PR DESCRIPTION
**Pull request by ohexter**
Allow all blind logos (subject to further discussion).

### CODE REVIEW

**Comment by Nanush7** [comment]
Yeah, we should decide if we want to allow logos for all blind events or just 3BLD.

File: wca-guidelines.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # <wca-title>WCA Guidelines
 
-<version>Version: January 1, 2024
+<version>Version: January 1, 2025
```
Comment (Nevseros) [inline wca-guidelines.md]:
```suggestion
<version>Version: January 1, 2024
```

File: wca-regulations.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # <wca-title>WCA Regulations
 
-<version>Version: January 1, 2024
+<version>Version: January 1, 2025
```
Comment (Nevseros) [inline wca-regulations.md]:
```suggestion
<version>Version: January 1, 2024
```

**Comment by CarterKoala** [comment]
Note B1+ will also need to be updated

File: wca-regulations.md [hunk 3]
```diff
@@ -150,8 +150,8 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 3k2) Penalty for attempts done with puzzles which are not permitted: disqualification of the attempt (DNF). The following exceptions apply:
         - 3k2a) If a non-permitted puzzle is found before a round is complete, affected results in the round may be replaced with extra attempts, at the discretion of the WCA Delegate.
         - 3k2b) 3x3x3 Multi-Blind: if any puzzles are found to be non-permitted, such puzzles may be individually counted as unsolved (without disqualifying the entire attempt), at the discretion of the WCA Delegate.
-- 3l) A puzzle may have one or more logos on a colored part. If it does, it must have at most one colored part with a logo or multiple logos. Exception: For blindfolded events, a puzzle must not have any logos.
-    - 3l1) The logo must be placed on a center piece. Exceptions for puzzles that do not have center pieces:
+- 3l) A puzzle may have one or more logos on a colored part. If it does, it must have at most one colored part with a logo or multiple logos. Exception: For 4x4x4 Blindfolded, a puzzle must not have any logos.
```
Comment (ohexter) [inline wca-regulations.md]:
```suggestion
- 3l) A puzzle may have one or more logos on a colored part. If it does, it must have at most one colored part with a logo or multiple logos. Exception: For 4x4x4 Blindfolded, a puzzle must not have any logos.
```

File: wca-regulations.md [hunk 4]
```diff
@@ -150,8 +150,8 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
+    - 3l1) The logo must be placed on a center piece. For 5x5x5 Blindfolded the logo must be placed on the most central center piece of a side. Exceptions for puzzles that do not have center pieces:
```
Comment (Nanush7) [inline wca-regulations.md]:
```suggestion
    - 3l1) The logo must be placed on a center piece. For 5x5x5 Blindfolded, the logo must be placed on the most central center piece of a side. Exceptions for puzzles that do not have center pieces:
```

**Pull request merged**